### PR TITLE
document gs mutation comparison component

### LINKED ISSUE #195: Document `gs-mutation-comparison-component`

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #195

### Summary

This introduces a template, because it turned out that documenting components mostly follows a common pattern. 

### Screenshot

grafik

### PR Checklist

- [x] All necessary documentation has been adapted.
- [x] The implemented feature is covered by an appropriate test.

### CODE REVIEW

File: components/tsconfig.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
         "outDir": "./build",
         "rootDir": "./",
         "strict": true,
-        "noUnusedLocals": true,
```
Comment (fengelniederhammer) [inline components/tsconfig.json:13]: This was redundant, eslint essentially has the same check. I needed to disable that check in some places for the type assertions.

File: components/src/utils/typeAssertions.ts
Comment (fengelniederhammer) [inline components/src/utils/typeAssertions.ts:1]: This was inspired by https://www.totaltypescript.com/how-to-test-your-types

File: components/src/web-components/PreactLitAdapter.tsx [hunk 2]
```diff
@@ -46,7 +46,6 @@ export abstract class PreactLitAdapter extends ReactiveElement {
     };
 
     override update(changedProperties: PropertyValues) {
-        console.log('this.lapis', this.lapis);
```
Comment (JonasKellerer) [inline components/src/web-components/PreactLitAdapter.tsx:49]: :)

**Review by JonasKellerer — APPROVED** [review]
Nice :)
LGTM

**Pull request merged**